### DISCUSSION TITLE
I flew in from ActivateCommandPalette!

### DISCUSSION THREAD

**Discussion by CoralPink** [wezterm/wezterm#3072] [General]
![discuss](https://user-images.githubusercontent.com/30016220/217440923-8e44037d-3dd1-4936-8f03-de931cb30118.png)

wonderful!

Please continue to entertain us. 🧸

**Comment by aorith** [comment]
I wonder if it can toggle an emit event that accepts user input (for example, to rename a tab or workspace)

**Comment by CoralPink** [comment]
Eh, wait!

Can `Emit event` registered in `keybind` be called from the palette as well? They were already registered ☺️

<img width="701" alt="emit" src="https://user-images.githubusercontent.com/30016220/217496189-2cf59ab6-ad9c-4420-9ebd-2db92a7fc790.png">

However, I thought it would be convenient if infrequently used items could be called from the palette without registering them in the key bindings 🐒

...Oh, I'm sorry. You were talking about parameters.😅
